Integreat/Integreat.Droid/Resources/values/styles.xml

```diff
@@ -24,6 +24,5 @@
     </style>
     
     <style name="SplashTheme.Base" parent="Theme.AppCompat.Light.NoActionBar">
-        <item name="android:windowBackground">@drawable/splash_screen</item>
```

Comment:
Splash screen is not compatible for some reason: https://github.com/bamlab/generator-rn-toolbox/issues/144

Reply:
maybe we can add this directly in code 🤔